starport/cmd/validation.go

```diff
@@ -0,0 +1,28 @@
+package starportcmd
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/starport/pkg/placeholder"
+)
+
+// FlagNoValidation disables validation.
+const FlagNoValidation = "no-validation"
+
+// RegisterValidationFlags adds validation specific flags to cobra.Command.
+func RegisterValidationFlags(cmd *cobra.Command) {
+	cmd.Flags().Bool(FlagNoValidation, false, "Disable validation.")
+}
+
+// WithValidation will configure validation based on the value of the flags.
+func WithValidation(ctx context.Context, cmd *cobra.Command) context.Context {
+	val, err := cmd.Flags().GetBool(FlagNoValidation)
+	if err != nil {
+		panic(err.Error())
```

Comment:
Why don;t we return the error in the function?

Reply:
i think that it can raise an error if the type doesn't match or if flag wasn't registered. actual validation of the value passed to a flag happens earlier

Reply:
we can infact ignore checking errors like this that can be only produced by programmer mistakes.